### ISSUE TITLE
excel readonly mode can't drag column wide

### ISSUE THREAD

**Issue by tengo100** [CollaboraOnline/online#8984]
### Describe the Bug
Excel readonly mode can't drag column wide, I want to widen the column to see the full content in the cell, How to solve it?

**Comment by Tex2002ans** [comment]
Hello @tengo100. Can you please fill out the "bug report" template?

This includes helpful information like:

- What exact Collabora version + OS you are using.
- Including screenshots/documents + an explanation of the issue.

- - -

If I understand you correctly, you might be using:

- A spreadsheet (ODS) + in "Read-Only Mode"

and you:

- Can't resize a column

That might be tricky, because read-only locks things down. But if you explain what you are trying to accomplish, maybe there's other solutions out there.

**Comment by tengo100** [comment]
thanks @Tex2002ans   

As shown below,  I want to widen the column to see the full content in the cell, how?  thanks.

<img width="1350" alt="image" src="https://github.com/CollaboraOnline/online/assets/18712660/a01cb0e3-cd54-4d1d-b161-93136433659b">

**Comment by Tex2002ans** [comment]
Hey @tengo100, thanks for the screenshot.

- - -

Now, if you could:

- Give your **Help > About** information.
- Attach a sample document.
- Give extra information like the "bug report" template asked for, like...
  - Steps to Reproduce
  - Expected Behavior
  - Actual Behavior
  - Desktop
  - [...]

that would really help a lot. :)

- - -

If we get a problem spreadsheet + you follow the submission template, it lets others replicate your problem! Then we could figure out exactly what's going wrong and get it fixed! :)

**Comment by fishstormX** [comment]
The same question with this

**Comment by timur-g** [comment]
I do not think this is possible. As changing column width would change the file also for other users.
